packages/flutter/lib/src/material/stepper.dart

```diff
@@ -232,6 +240,27 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// give background color to header
+  final Color? backgroundHeaderColor;
+
+  /// set the BoderRadius to header
+  final BorderRadiusGeometry? headerRadius;
+
+  /// set the color to text of active step
+  final Color? activeCircleTextColor;
```

Comment:
I can't seem to figure out how to get this color to impact how a Stepper draws. Can you provide an example in the docstring?

Reply:
Update: I realized I did end up figuring this out, but never updated the comment. (Spoiler: I didn't have the requisite step in the correct state.) That said, I would still like to see all of the docstrings fleshed out a bit more :)